Sources/Subs-Auth.php

```diff
@@ -809,8 +809,8 @@ function rebuildModCache()
 }
 
 /**
- * The same thing as setcookie but gives support for HTTP-Only cookies in PHP < 5.2
- * @todo We can remove this since SMF requires PHP >= 5.3.8 now
```

Comment:
I was fixing the comment to be more accurate. This used to actually rebuild the cookie by hand rather than just using setcookie.